<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plottingAndBackend1Class</class>
 <widget class="QMainWindow" name="plottingAndBackend1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>plottingAndBackend1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ECG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Electrode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Pacing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="plottingAndBackend1.qrc"/>
 </resources>
 <connections/>
</ui>
